Sources/PostgreSQL/Connection.swift

```diff
@@ -171,6 +180,37 @@ public final class Connection: ConnInfoInitializable {
         }
     }
     
+    /// Creates a dispatch read source for this connection that will call `callback` on `queue` when a notification is received
+    ///
+    /// - Parameter channel: the channel to register for
+    /// - Parameter queue: the queue to create the DispatchSource on
+    /// - Parameter callback: the callback
+    /// - Parameter note: The notification received from the database
```

Comment:
The `note` and `err` parameters cannot be commented like this. It should be part of the `callback` comments. Also I would write out their full names instead of abbreviations.

Reply:
Why? If you comment them that way, option clicking on the listen function shows them in a nested table underneath the callback parameter.

Reply:
Ok sorry, didn’t know it worked like that.